src/components/dashboard/Dashboard.js

```diff
@@ -114,13 +114,6 @@ const Dashboard = props => {
   const { avatar, fullName } = gdstore.get('profile')
   const [feeds, setFeeds] = useState([])
   const [headerLarge, setHeaderLarge] = useState(true)
-  const scale = {
```

Comment:
This shouldn't be removed

Reply:
not used, i comment this?

Reply:
It was being used on the Animation view that wrapped the ClaimButton